docs/platform/developing/flashing-rpi.md

```diff
@@ -0,0 +1,511 @@
+# Flashing an Image to RPI
+
+## Introduction
+
+This topic describes how to flash Tizen on SD card with or without IoT Setup Manager and setting up Raspberry Pi 3 or 4.
+
+## Supported systems
+
+|Division	|Ubuntu	|Windows	|
+|:---:		|:---:	|:---:		|
+|Headless 32-bit|Yes	|No			|
+|Headed 32-bit|Yes	|Yes		|
+|Headed 64-bit|Yes	|No			|
+
+> **Note**
+>
+> In windows, only EXT4 type file of rootfs.img is supported to flash rpm bundles. (While using Headed 64 bit image and Headless 32bit image, RPMs cannot be installed through IOT setup manager because rootsfs.img is of BTRFS type file.)
```

Comment:
Global: Please use the updated Note tag.

> [!Note]
> In Windows, only EXT4 type file of rootfs.img is supported to flash rpm bundles. If you are using Headless and Headed 64-bit image, RPMs cannot be installed through IOT setup manager because rootsfs.img is of BTRFS type file.

Reference: https://github.com/Samsung/tizen-docs/pull/1293#discussion_r509322141